components/nexus-common/src/main/java/org/sonatype/nexus/common/entity/Entity.java

```diff
@@ -24,6 +24,11 @@
   private EntityMetadata metadata;
 
   @Nullable
+  public EntityId id() {
+    return getEntityMetadata() != null ? getEntityMetadata().getId() : null;
+  }
```

Comment:
Why?  Entity was meant to not expose helpers like this.  But if you wanted then getEntityId() would be better than id() to keep the namespace relativly clean for the sub-class to do what it wants.